includes/Admin/Status/Status.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace WeDevs\Dokan\Admin\Status;
+
+use Exception;
+use WeDevs\Dokan\Abstracts\StatusElement;
+use WeDevs\Dokan\VendorNavMenuChecker;
+
+class Status extends StatusElement {
+
+    protected bool $support_children = true;
+    protected string $hook_key = 'dokan_status';
+
+    public function __construct() {
+        parent::__construct( 'dokan-status' );
+
+        try {
+            $this->describe();
+        } catch ( Exception $e ) {
+            dokan_log( $e->getMessage() );
+        }
+
+        add_action( 'dokan_admin_menu', [ $this, 'register_menu' ], 99, 2 );
+        add_action( 'dokan_register_scripts', [ $this, 'register_scripts' ] );
+        add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
+    }
+
+    /**
+     * Get a new Page object.
+     *
+     * @param string $id ID.
+     *
+     * @return Page
+     */
+    public static function page( string $id ): Page {
+        return new Page( $id );
+    }
+
+    /**
+     * Get a new tab object.
+     *
+     * @param string $id ID.
+     *
+     * @return Tab
+     */
+    public static function tab( string $id ): Tab {
+        return new Tab( $id );
+    }
+
+    /**
+     * Get a new Section object.
+     *
+     * @param string $id ID.
+     *
+     * @return Section
+     */
+    public static function section( string $id ): Section {
+        return new Section( $id );
+    }
+
+    /**
+     * Get a new SubSection object.
+     *
+     * @param string $id ID.
+     *
+     * @return SubSection
+     */
+    public static function sub_section( string $id ): SubSection {
+        return new SubSection( $id );
+    }
+
+    /**
+     * Get a new Table object.
+     *
+     * @param string $id ID.
+     *
+     * @return Table
+     */
+    public static function table( string $id ): Table {
```

Comment:
- Pls use factory 
- Change the visibility to `protected` and remove the `static` so that it can only be used from the self & descendants.

CC: @shohag121 vai.